### DISCUSSION TITLE
How to configure `pageInfo` to hide reading time?

### DISCUSSION THREAD

**Discussion by YconquestY** [vuepress-theme-hope/vuepress-theme-hope#2184] [Q&A] [answered]
I see in the [doc](https://vuepress-theme-hope.github.io/v2/config/theme/layout.html#pageinfo) that I may opt not to provide reading time estimation. My configuration is as follows
```js
const { hopeTheme } = require("vuepress-theme-hope")

…

module.exports = {
    …
    theme: hopeTheme({
        …
        title: {
            pageInfo: ["Author", "Date", "Original", "Category", "Tag", false, false, false]
        }
    })
    …
}
```
This does not work: `reading time` still exists. May I know how to correctly configure `ArticleInfo[]`?

**Comment by Mister-Hope** [comment, marked as answer]
Do not add those `false`, and we do not have `title.pageInfo`, those headings are only for grouping, so you should write:

```js
export default {
    …
    theme: hopeTheme({
        …
        pageInfo: ["Author", "Date", "Original", "Category", "Tag"]
    })
    …
}
